Libs/Optimize/OptimizeParameterFile.cpp

```diff
@@ -918,7 +922,7 @@ int OptimizeParameterFile::get_num_inputs(TiXmlHandle* docHandle)
     throw std::invalid_argument(message);
   }
 
-  return inputs.size() / domains_per_shape;
+  return inputs.size(); // domains_per_shape;
```

Comment:
Is this intentional?  Changing the output of get_num_inputs to be the total number of domains rather than the number of samples?

Reply:
Yes, this is what fixes Penny's issue. This function is only used for the constraints input functions, so it doesn't affect anything else. Also, you should expect get_num_ inputs to return the number of inputs (as enclosed in the input tabs in the xml) and not the number of shapes.

Reply:
Sounds good.  Let's get rid of the comment "// domains_per_shape;"